source/MechEngineer.csproj

```diff
@@ -130,13 +98,43 @@
     <Compile Include="MechEngineerSettings.cs" />
     <Compile Include="Patches\MechStatisticsRulesCalculateHeatEfficiencyStatPatch.cs" />
     <Compile Include="Extensions.cs" />
+    <Compile Include="UniqueItemStuff.cs" />
   </ItemGroup>
   <ItemGroup>
     <None Include="..\mod.json" />
     <None Include="..\README.md" />
     <None Include="..\Settings.json" />
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Reference Include="0Harmony">
+      <HintPath>D:\Steam\steamapps\common\BATTLETECH\BattleTech_Data\Managed\0Harmony.dll</HintPath>
```

Comment:
Create file called MechEngineer.csproj.user, put that inside:
```
<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 <PropertyGroup>
    <BattleTechGame>D:\Games\GalaxyClient\Games\BATTLETECH</BattleTechGame>
  </PropertyGroup>
</Project>
```

obvisouly your path to the game.

Then replace all the paths in here with $(BattleTechDir)